stl/inc/bit

```diff
@@ -127,21 +127,9 @@ extern int __isa_available;
 }
 
 template <class _Ty>
-_NODISCARD int _Checked_x86_x64_countl_zero(const _Ty _Val) noexcept {
-    constexpr int _Digits  = numeric_limits<_Ty>::digits;
-
-#ifndef __AVX2__
-    const bool _Have_lzcnt = __isa_available >= __ISA_AVAILABLE_AVX2;
-    // lzcnt (when it doesn't fall back to bsr) is defined correctly for zero
-    // bsr has undefined output for zero
-    if (!_Have_lzcnt && _Val == 0) {
-        return _Digits;
-    }
-#endif // __AVX2__
+_NODISCARD int _x86_x64_countl_zero_lzcnt(const _Ty _Val) noexcept {

```

Comment:
This `_underscore_lowercase` identifier isn't `_Ugly` enough according to [the rules](https://eel.is/c++draft/lex.name#3). Since `_X86_x64` would look weird, how about rearranging these names to:

* `_Countl_zero_lzcnt_x86_x64` and `_Countl_zero_bsr_x86_x64`
* `_Countl_zero_x86_x64_lzcnt` and `_Countl_zero_x86_x64_bsr`
* `_Unchecked_x86_x64_countl_zero_lzcnt` and `_Unchecked_x86_x64_countl_zero_bsr` (following the example of their caller `_Checked_x86_x64_countl_zero`)
* Something else?

Reply:
Any of these will do. `_Unchecked` is slightly confusing for `bsr` though. Maybe just:
 * `_Countl_zero_lzcnt` and `_Countl_zero_bsr`

`lzcnt` implies `x86_x64` and for `bsr` it may be a good idea to use it on ARM either.